helper_lib.sh

```diff
@@ -77,7 +77,16 @@ get_docker_cumulative_command_line_args() {
 # Does not account for option default or implicit options.
 get_docker_effective_command_line_args() {
     OPTION="$1"
-    get_docker_cumulative_command_line_args $OPTION | tail -n1
+    get_docker_cumulative_command_line_args "$OPTION" | tail -n1
+}
+
+get_docker_configuration_file_args() {
+    OPTION="$1"
+    FILE="$(get_docker_effective_command_line_args '--config-file' | \
```

Comment:
I don't think this handles the case where `--config-file` is not explicitly set, and the [default value](https://docs.docker.com/engine/reference/commandline/dockerd/#/options) of `/etc/docker/daemon.json` is being used.